lib/v2/screens/dashboard/interactor/usecases/load_token_balances_use_case.dart

```diff
@@ -4,9 +4,9 @@ import 'package:seeds/v2/datasource/remote/model/token_model.dart';
 
 export 'package:async/src/result/result.dart';
 
-class LoadBalanceUseCase {
-  Future<Result> run(TokenModel token) {
+class LoadTokenBalancesUseCase {
+  Future<List<Result>> run(List<TokenModel> tokens) {
     var account = settingsStorage.accountName;
-    return BalanceRepository().getTokenBalance(account, token);
+    return BalanceRepository().getTokenBalances(account, tokens);
```

Comment:
the function that was created in the repo belongs here.

Reply:
Done